models/usersModel.js

```diff
@@ -0,0 +1,55 @@
+const db = require("../db/connection");
+
+function insertNewUser(email, password, role) {
+  const token = "12345";
+  return db
+    .query(
+      `INSERT INTO users(email,password,role,isVerified,token)
```

Comment:
You do not need to be writing query yourself. There is a library you can use which is https://sequelize.org/

Also, you do not need to write create table query. The sequelize library will handle that for you.

Read about ORM (Object Relational Mapping)